hack/test-operator-integration.sh

```diff
@@ -4,4 +4,5 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+rm -rf ./test-output
```

Comment:
why not to add the `clean` target to https://github.com/openshift/multi-operator-manager/blob/3db370e102a9182e54ae8cf88002f575e2823b96/Makefile#L46 target ?